test/stdtypes/Makefile

```diff
@@ -34,4 +34,5 @@ regenerate:
 	Mgoogle/protobuf/timestamp.proto=github.com/gogo/protobuf/types\
 	:. \
 	--proto_path=../../../../../:../../protobuf/:. stdtypes.proto
+	go test -race .
```

Comment:
Lets add a separate test target to this Makefile 
```
test:
  go test -race -run= TestConcurrentTextMarshal .
```

And then we can add this to the top Makefile
https://github.com/gogo/protobuf/blob/master/Makefile#L117-L119

```
tests:
  go build ./test/enumprefix
  go test ./...
  (cd test/stdtypes && make test)
```